src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -3026,6 +3027,32 @@ public static void DCS_BasicRoundtripDCRVerifyWireformatScenarios()
     }
 
     [Fact]
+    public static void DCS_BasicRoundtripDCRDelegates()
+    {
+        var dataContractSerializerSettings = new DataContractSerializerSettings()
+        {
+            DataContractResolver = new VerySimpleResolver(),
+            IgnoreExtensionDataObject = false,
+            KnownTypes = null,
+            MaxItemsInObjectGraph = int.MaxValue,
+            PreserveObjectReferences = false
```

Comment:
I think it'd be better to keep just the necessary settings here so that the purpose of the test is clear.